src/BatteryStateDisplayService.cpp

```diff
@@ -51,7 +51,8 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
+    batteryStateOfChargeService_.isCharging();
+    qDebug()<<"Time finish charging: " <<batteryStateOfChargeService_.timeWhenChargedOrDepleted();
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
```

Comment:
So at that, you would want to convert the QTime into a QString using the `toString()` function. I think that's the name of the function, but the docs are down so I'm not sure ;o